client/src/main/kotlin/no/nav/su/se/bakover/client/dokarkiv/Journalpost.kt

```diff
@@ -50,22 +51,22 @@ sealed class Journalpost {
                     DokumentVariant.OriginalJson(
                         fysiskDokument = Base64.getEncoder()
                             .encodeToString(objectMapper.writeValueAsString(this).toByteArray()),
-                    )
-                )
-            )
+                    ),
+                ),
+            ),
         )
     }
 
-    data class Vedtakspost(
+    data class Vedtakspost private constructor(
         val person: Person,
         val saksnummer: Saksnummer,
-        val brevInnhold: BrevInnhold,
-        val pdf: ByteArray
+        val pdfRequestJson: String,
```

Comment:
Antar dette er original-dataene som ble brukt for å generere PDFen. Kan kanskje kalle den originalJson. Det at vi har trukket PDF-greiene ut i en egen tjeneste (slik at det går en http request), bør ikke påvirke navngivinga.